### PR TITLE
BO-309: integration guide for my account section in merchant po…

### PR DESCRIPTION
**Pull request by tobi812**
…rtal

## PR Description

Integration guide for My Account section in the merchant portal

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

File: docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-my-account-feature-integration.md [hunk 1]
```diff
@@ -0,0 +1,225 @@
+---
+title: Merchant Portal My Account feature integration
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-my-account-feature-integration.md:2]: We have IGs for Spryker features, not for epic. the corresponding IGs should be updated

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md [hunk 2]
```diff
@@ -553,3 +554,145 @@ The following page should now show the login page for MerchantPortal: `https://y
...
+{
+    /**
+     * @param string[][] $installerRules
+     *
+     * @return string[][]
+     */
+    protected function addMerchantPortalInstallerRules(array $installerRules): array
+    {
+        $bundleNames = [
+            'user-merchant-portal-gui',
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md:583]: please add other modules from this feature

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md [hunk 3]
```diff
@@ -553,3 +554,145 @@ The following page should now show the login page for MerchantPortal: `https://y
...
+    <security-merchant-portal-gui>
+        <label>Logout</label>
+        <title>Logout</title>
+        <icon>logout</icon>
+        <bundle>security-merchant-portal-gui</bundle>
+        <controller>logout</controller>
+        <action>index</action>
+    </security-merchant-portal-gui>
+</config>
+```
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md:662]: not needed

File: docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md [hunk 4]
```diff
@@ -553,3 +554,145 @@ The following page should now show the login page for MerchantPortal: `https://y
...
+
+```php
+<?php
+
+namespace Pyz\Zed\MerchantUser;
+
+use Generated\Shared\Transfer\RuleTransfer;
+use Spryker\Zed\MerchantUser\MerchantUserConfig as SprykerMerchantUserConfig;
+
+class MerchantUserConfig extends SprykerMerchantUserConfig
```
Comment (ilyakubanov) [inline docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md:611]: this is not correct anymore, please check AclMerchantPortalConfig

Comment (ilyakubanov) [reply docs/marketplace/dev/feature-integration-guides/202108.0/marketplace-merchant-portal-core-feature-integration.md]: @tobi812 AclMerchantPortalConfig should be added

**Review by ilyakubanov — APPROVED** [review]

**Review by gibmirmilch — APPROVED** [review]

**Pull request merged**